frontend/src/pages/404ErrorPage.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    Box,
+    Button,
+    Flex,
+    Text,
+    Link as LinkButton,
+    IconButton
+} from "@radix-ui/themes";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import shrugBro from "../../src/assets/Shrug-bro.svg"
+import { LuGithub } from "react-icons/lu"
+
+const NotFoundPage = () => {
+
+    const navigate = useNavigate();
+    const goToLoginPage = () => {
+        navigate("/login");
+    }
+
+    const goToChannelPage = () => {
+        navigate("/channel")
+    }
+
+    return (
+        <Flex direction="column" align="center" gap="1" justify="center">
+            <Flex direction="column" align="center" gap="1">
+                <Box className="w-[30vw] max-sm:w-[70vw]">
+                    <img src={shrugBro} ></img>
+                </Box>
+
+                <Text className="cal-sans" align="center" size="9"> 404 Error </Text>
+                <Text className="cal-sans text-[3vw] max-sm:text-[5vw]" align="center">  Seems Like you have wondered too far ... </Text>
+
+                <Flex direction="column" align="center" gap="3">
+                    <Button className="w-[30vw] h-[5vh] max-sm:w-[60vw]" variant="soft" onClick={goToChannelPage}> BACK TO CHANNELS </Button>
```

Comment:
The button text should not be all caps. 
This styling is unnecessary, just size='1' or size='2' should do, if you want the button width to be similar to that of the button content set width='fit-content'.
The button text can be 'Go Back'